src/Actions/index.js

```diff
@@ -0,0 +1,12 @@
+export function getEvents() {
+  return dispatch => {
+    return fetch('http://localhost:4000/api/events')
```

Comment:
I believe this will need to be an ENV variable so that when this gets pushed to production it will hit the heroku site.

Reply:
Cool lets go over that tomorrow for standup.